submissions/5mountains/popup/css/main.css

```diff
@@ -0,0 +1,260 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    list-style-type: none;
```

Comment:
Let's add this `list-style-type` for resetting styles for `ul` (not to the all elements)